src/components/OfflineWithFeedback.js

```diff
@@ -87,6 +87,7 @@ const OfflineWithFeedback = (props) => {
         .keys()
         .sortBy()
         .map(key => props.errors[key])
+        .uniq()
```

Comment:
I added `uniq` here since we're passing the errors from both customUnits and customUnitRates [here](https://github.com/Expensify/App/pull/10557/files#diff-9d527d65213f52fe1136ce113e7601e11bef8a50a22e0472786406a13a81faefR194-R197), which could potentially be identical because they're both set to the same string [here](https://github.com/Expensify/App/pull/10557/files#diff-5fd7c80f3a9b43595640753f4037ca2b7d0bebeca557a20c4a06f89dc5028ccfR647-R649) and [here](https://github.com/Expensify/App/pull/10557/files#diff-5fd7c80f3a9b43595640753f4037ca2b7d0bebeca557a20c4a06f89dc5028ccfR713-R715).